package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@codefresh-io/service-base",
-  "version": "6.0.0",
+  "version": "6.0.2-beta",
```

```suggestion
  "version": "7.0.0",
```
I suggest bumping major version since exported mongoclient has breaking changes.

---

yeah sure this is temporary, I will change before merging